datadog/api/metadata.py

```diff
@@ -0,0 +1,60 @@
+# datadog
+from datadog.api.resources import GetableAPIResource, CreateableAPIResource, \
+    UpdatableAPIResource, ActionAPIResource
+
+
+class Metadata(GetableAPIResource, CreateableAPIResource, UpdatableAPIResource, ActionAPIResource):
```

Comment:
I added some clarification in the update docstring. The update request will create the metadata if it doesn't exist.